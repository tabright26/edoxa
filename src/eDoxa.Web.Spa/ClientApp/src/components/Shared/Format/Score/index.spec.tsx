import React from "react";
import renderer from "react-test-renderer";
import { Score } from ".";

it("renders without crashing", () => {
  const tree = renderer.create(<Score score={100} />).toJSON();
  expect(tree).toMatchSnapshot();
});
