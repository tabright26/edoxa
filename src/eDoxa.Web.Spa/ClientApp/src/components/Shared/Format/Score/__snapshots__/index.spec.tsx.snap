// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<span>
  100.00
</span>
`;
